db/migrate/20200514214910_rename_bookmarks_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RenameBookmarksController < ActiveRecord::Migration[5.2]
+  def up
+    original_controller = 'foreman_tasks_tasks'
+    original_bookmarks = Bookmark.where({ :controller => original_controller })
+    original_bookmarks_names = original_bookmarks.map { |b| b.name }
+    
+    mistake_controller = 'foreman_tasks/tasks'
+    mistake_bookmarks = Bookmark.where({ :controller => mistake_controller })
+    mistake_bookmarks.each do |b|
```

Comment:
I know it's not important here, since you won't have too much records, but it makes sense to use [`#find_each`](
https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) method for iteration over database results:
```suggestion
    mistake_bookmarks.find_each do |b|
```